spec/reports/github_api_client_post_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "sinatra/base"
+require "webmock/rspec"
+require "reports/github_api_client"
+
+class FakeGitHub < Sinatra::Base
+  attr_reader :gists
+
+  def initialize
+    super
+    @gists = []
+  end
+
+  post '/gists' do
+    content_type :json
+    payload = JSON.parse(request.body.read)
+    if payload["files"].any? { |name, hash| hash["content"] == "" }
```

Comment:
Unused block argument - `name`. If it's necessary, use `_` or `_name` as an argument name to indicate that it won't be used.